public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -31,6 +33,11 @@ export class AppChromeService {
     chromeless: true, // start out hidden to not flash it on pages without chrome
     sectionNav: { node: { text: t('nav.home.title', 'Home') }, main: { text: '' } },
     searchBarHidden: store.getBool(this.searchBarStorageKey, false),
+    megaMenu:
+      config.featureToggles.dockedMegaMenu &&
+      JSON.parse(window.localStorage.getItem(DOCKED_LOCAL_STORAGE_KEY) ?? 'false')
```

Comment:
could we move localStorage retrieval (and setting) to some small helper functions, just to avoid the `window.localStorage` directly in the compone itself?

Reply:
turns out we already have `store.set`/`store.getBool` which basically handles this logic 🥳